### PR TITLE
feat: provide modern normalize function

### PR DESCRIPTION
**Pull request by chambo-e**
## Type

- Feature

### Summarise concisely:

This is to avoid to some magic fuckery we started doing since the removal of GlobalStyle and have a shared ground of css development 

## Relevant logs

| Page |   Before   |      After |
| :--- | :--------: | ---------: |
| url  | Screenshot 2021-12-21 at 17 33 18 | Screenshot 2021-12-21 at 17 33 13
| url  | screenshot | screenshot |

### CODE REVIEW

**Review by vincentaudebert — APPROVED** [review]
good idea 👍

**Comment by vincentaudebert** [comment]
Screenshot 2021-12-21 at 18 22 15

Not linked to ticket but it looks like textbox and richselect doesn't have the same disabled colors 😅

File: src/components/Button/__stories__/index.stories.tsx [hunk 1]
```diff
@@ -141,23 +141,17 @@ Action.parameters = {
 }
 Action.decorators = [
   () => (
-    <>
+    <div style={{ display: 'flex', flexWrap: 'wrap', gap: '16px' }}>
```
Comment (QuiiBz) [inline src/components/Button/__stories__/index.stories.tsx:144]: I think this is happening a few times in scaleway-ui, shouldn't we replace all mr/mb with a flex gap on the parent? (in a separate PR)

**Review by QuiiBz — APPROVED** [review]

File: src/components/Button/__stories__/index.stories.tsx (hunk 1 above)
Comment (chambo-e) [reply src/components/Button/__stories__/index.stories.tsx:144]: Yes absolutely, we are in the process of removing xstyled props, so whenever you can please do :)

**Pull request merged**